vita3k/renderer/src/texture_cache.cpp

```diff
@@ -505,6 +505,7 @@ void upload_bound_texture(const TextureCacheState &cache, const SceGxmTexture &g
         case SCE_GXM_TEXTURE_CUBE_ARBITRARY:
             width = next_power_of_two(width);
             height = next_power_of_two(height);
+            break;
```

Comment:
Remove this this change, is wrong

Reply:
Maybe add instead:
```cpp
[[fallthrough]];
```
https://en.cppreference.com/w/cpp/language/attributes/fallthrough

Reply:
no, here this code is just if is this code, set this, and continue

Reply:
@Macdu, Do you use [[attributes]] in this project?
I saw code another project where almost every function had [[nodiscard]] and header file was so big